spark-dependencies/pom.xml

```diff
@@ -36,7 +37,7 @@
 
 
   <properties>
-    <spark.version>1.4.1</spark.version>
+    <spark.version>1.6.1</spark.version>
```

Comment:
Thanks @shijinkui for the contribution. I want to point out that change spark.version here is probably not the good way to make 1.6.1 as a default build. while all other dependency versions are remained for spark 1.4.1.

I think activating profile `spark-1.6` by default, could be a possible way to make 1.6.1 default.